windows/window_manager.cpp

```diff
@@ -58,6 +58,9 @@ class WindowManager {
 
   bool is_resizing_ = false;
   bool is_moving_ = false;
+  bool is_active_ = true;
+  bool is_hide_on_deactivate_ = true;
+  bool is_hidden_on_blur = false;
```

Comment:
I see that most variables on this file use `this_naming_convention_`. Can we this variable follow it to?

Reply:
Done.